src/propTypes.js

```diff
@@ -312,26 +312,13 @@ export const experimentResultPropTypesShape = PropTypes.shape({
     description: PropTypes.string,
     filename: PropTypes.string,
     file_format: PropTypes.oneOf([
-        "SAM",
```

Comment:
this change is incorrect - it should match the schema in katsu

Reply:
Thanks for the correction!